pkgs/development/python-modules/smhi/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, aiohttp
+, black
+, coverage
+, flake8
+, mock
+, pyflakes
+, pylint
+, pytest
+, pytest-asyncio
+, pytest-cov
+, requests
+, requests-toolbelt
+}:
+
+buildPythonPackage rec {
+  pname = "smhi";
+  version = "1.0.13";
+
+  src = fetchPypi {
```

Comment:
The source tarball published on PyPI doesn't contains the tests but the source from GitHub does. It looks like that it will not be possible to run the tests as they want to communicate with `opendata-download-metfcst.smhi.se:443`.

If that's the case add `doCheck = false;` and remove all test related inputs.

Reply:
Yes, tests communicate with that url